include/crossplatform_shared_ptr.h

```diff
@@ -20,8 +20,8 @@
         using std::shared_ptr;
     #endif
 #elif defined(__ISLINUX__) // GCC
-    #include <tr1/memory>
-    using namespace std::tr1;
+    #include <memory>
```

Comment:
Are these changes necessary?  Until we switch the full codebase to C++11, I don't want to mess with this.

Reply:
I could not get the code to compile without making this change. I am on Ubuntu Linux (18.04.3 LTS) with GCC 7.4.0. Should we include an if statement like was done for MacOS that first checks if <tr1/memory> can be included?

Reply:
I think you are right that on linux this isn't too serious, and should work.  This was for monkey-patching ancient versions of c++ compilers, shared_ptr is in the standard library now.